README.md

```diff
@@ -1,393 +1,296 @@
-{
-  "browsers" : [ "firefox", "safari" ]
-}
-```
+**Note:** Depending on your OS, the target browsers should not already be open/running before using this library. Electron and PhantomJS are the exceptions, as they can always be started multiple times.
 
-#### Browserstack
+#### `browsers`
+Type: `Array`  
+Default value: `['phantom']`  
+The browsers that will be used to run tests. For local browsers, use a browser name string (see [launchpad](https://npmjs.com/launchpad) for more info). For a remote/BrowserStack browser, use a [browser object](https://npmjs.com/browserstack#browser-objects). Mixing both types is not currently possible.
 
-Browserstack hosts virtual machines running specific versions of web browsers. It is an extremely useful
-tool for cross-browser testing running a remote desktop in you browser. To use BrowserStack via the configuration
-API you need to provide a username and password:
+#### `launch`
+Type: `Object`  
 
-```js
-{
-  "launch": {
-    "type": "browserstack",
-    "username": "your browserstack username",
-    "password": "your browserstack password",
-    "version": "browserstack API version (recommended: 4)"
-  }
-}
-```
+#### `launch.password`
+Type: `String`  
+Default value: `undefined`  
+Your BrowserStack API key.
 
-To start a worker, you must provide a valid [browser object](https://github.com/scottgonzalez/node-browserstack#browser-objects) to the `browser` option:
+#### `launch.type`
+Type: `String`  
+Default value: `'local'`  
+The test environment. Possible values are `'local'` and `'browserstack'`.
 
-```js
-{
-  browsers: [{
-    os: "win",
-    os_version: "7.0",
-    browser: "ie",
-    browser_version: "8.0"
-  }, {
-    os: "win",
-    os_version: "7.0",
-    browser: "ie",
-    browser_version: "11.0"
- }]
-}
-```
+#### `launch.username`
+Type: `String`  
+Default value: `undefined`  
+Your BrowserStack username.
 
-An example configuration that runs your tests on an iPad Mini and Samsung Galaxy S3 emulator using BrowserStack
-in a CI environment (outputting XUnit logs) could look like this:
+#### `launch.version`
+Type: `String`  
+Default value: `undefined`  
+The BrowserStack API version you'd like to use. The recommended value is `4`.
 
-```json
-{
-    "reporter" : "XUnit",
-    "tunnel": {
-      "type": "browserstack",
-      "key": "your browserstack key"
-    },
-    "launch": {
-      "type": "browserstack",
-      "username": "your browserstack username",
-      "password": "your browserstack password",
-      "version": 4
-    },
-    "browsers": [{
-      "os": "ios",
-      "device": "iPad Mini",
-      "os_version": "6.0"
-    }, {
-      "os": "android",
-      "device": "Samsung Galaxy S III",
-      "os_version": "4.1"
-    }]
-  }
-}
-```
 
 ### Code Coverage
 
-The `coverage` options is used to instrument and report code coverage using [Istanbul](https://github.com/gotwarlost/istanbul). There are several options available for configuring how code coverage is reported:
```

Comment:
The removed example is in the Configuration API area. The mention of Istanbul might be useful to keep, though.